tests/timeclock.test

```diff
@@ -18,11 +18,41 @@ o 2009/1/3 09:00:00
 2009-01-03 * and a description
     (some:account name)           1.00h
 
->>>2
->>>= 0
+>2
+>= 0
+
+# For a missing clock-out, now is implied
+<
+i 2020/1/1 08:00
+$ hledger -f - balance
+> /./
+>= 0
+
+# For a log starting not with clock-out, print error
+<
+o 2020/1/1 08:00
+$ hledger -f - balance
+>2 /line 1: expected timeclock code i/
+>= !0
+
+# For a different log starting not with clock-out, print error
+<
+o 2020/1/1 08:00
+o 2020/1/1 09:00
+$ hledger -f - balance
```

Comment:
How is this one different from the previous ? Should it be testing i o o instead ?

Reply:
The previous test is different because a log with only one entry is a special case.

Reply:
We could test i o o or i o o o, but I would provide a minimal example as input.